coalib/settings/Section.py

```diff
@@ -14,6 +15,7 @@ def append_to_sections(sections,
                        key,
                        value,
                        origin,
+                       to_append=False,
```

Comment:
this is backwards incompatible.
It make it backwards compatible, move the new arg to the end of the args.